published-namespaces.txt

```diff
@@ -240,6 +240,50 @@ AndroidX.ExifInterface.Media
 AndroidX.Fragment.App
 AndroidX.Fragment.App.StrictMode
 AndroidX.GridLayout.Widget
+AndroidX.Health.Connect.Client
+AndroidX.Health.Connect.Client.Aggregate
+AndroidX.Health.Connect.Client.Changes
+AndroidX.Health.Connect.Client.Contracts
+AndroidX.Health.Connect.Client.DataNotification
+AndroidX.Health.Connect.Client.Impl
+AndroidX.Health.Connect.Client.Impl.Converters.Aggregate
+AndroidX.Health.Connect.Client.Impl.Converters.Changes
+AndroidX.Health.Connect.Client.Impl.Converters.Datatype
```

Comment:
Looks like Google is going with `DataType` capitalization here.

Source: https://developer.android.com/reference/androidx/health/services/client/data/DataType

Reply:
fixed